### PR TITLE
rework zed to jive with v1 APIs

### PR DESCRIPTION
**Pull request by jzelinskie**
As we work on fleshing out a stable v1 API, zed is the first test bed to adopt it.
This PR is what migrates it to using all of the concepts and APIs for v1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,10 +80,10 @@ For each type of noun used in Authzed, there is a zed subcommand:
 For example, you can read Object Definitions in a Permissions System's Schema, check permissions, and even create or delete relationships.
 
 ```sh
-zed schema read user document
-zed permission check user:emilia writer document:firstdoc
-zed relationship create user:beatrice reader document:firstdoc
-zed relationship delete user:beatrice reader document:firstdoc
+zed schema read
+zed permission check document:firstdoc writer user:emilia
```
Comment (josephschorr) [inline README.md:84]: Ideally, these should be checking permissions, so `read` and `write` (even though it works for both, of course)

File: README.md [hunk 2]
```diff
@@ -94,20 +93,20 @@ Experimentally, zed embeds an instance of [OPA] that supports additional builtin
 The following functions have been added:
 
 ```rego
-authzed.check("subject:id", "permission", "object:id", "zedtoken")
+authzed.check("object:id", "permission", "subject:id", "zedtoken")
```
Comment (josephschorr) [inline README.md:96]: `resource:id`?

File: cmd/zed/experiment.go [hunk 3]
```diff
@@ -65,53 +65,58 @@ func registerAuthzedBuiltins(system string, client *authzed.Client) {
...
-				User: &v0.User{UserOneof: &v0.User_Userset{Userset: &v0.ObjectAndRelation{
-					Namespace: stringz.Join("/", system, subjectNS),
-					ObjectId:  subjectID,
-					Relation:  subjectRel,
-				}}},
 			}
 
 			if zedtoken != "" {
-				request.AtRevision = &v0.Zookie{Token: zedtoken}
+				request.Consistency = &v1.Consistency{
```
Comment (josephschorr) [inline cmd/zed/experiment.go:108]: Do we want to allow an optional flag to set this to other consistency options?

File: cmd/zed/permission.go [hunk 4]
```diff
@@ -41,7 +42,7 @@ var permissionCmd = &cobra.Command{
 }
 
 var checkCmd = &cobra.Command{
-	Use:               "check <subject:id> <permission> <object:id>",
+	Use:               "check <object:id> <permission> <subject:id>",
```
Comment (josephschorr) [inline cmd/zed/permission.go:45]: `resource`

File: cmd/zed/permission.go [hunk 5]
```diff
@@ -106,27 +107,29 @@ func checkCmdFunc(cmd *cobra.Command, args []string) error {
...
+				ObjectId:   subjectID,
 			},
-		}},
+			OptionalRelation: subjectRel,
+		},
 	}
 
-	if zedToken := cobrautil.MustGetString(cmd, "revision"); zedToken != "" {
-		request.AtRevision = &v0.Zookie{Token: zedToken}
+	if zedtoken := cobrautil.MustGetString(cmd, "revision"); zedtoken != "" {
```
Comment (josephschorr) [inline cmd/zed/permission.go:125]: Maybe we extract this into a util method and use everywhere?

Comment (jzelinskie) [reply cmd/zed/permission.go:127]: I think I'll just create an issue for "adding a consistency flag" and we can do that in a follow up.

**Review by ecordell — APPROVED** [review]
LGTM

**Pull request merged**